calc/math/types/fixed_point.rs

```diff
@@ -2,9 +2,8 @@ use decimal::*;
 #[allow(unused_imports)]
 use gstd::ToString;
 use gstd::{Decode, Encode, TypeInfo};
-
-#[decimal(12)]
+#[decimal(12, U192T)]
 #[derive(Default, Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Encode, Decode, TypeInfo)]
 #[codec(crate = gstd::codec)]
 #[scale_info(crate = gstd::scale_info)]
-pub struct FixedPoint(pub u128);
+pub struct FixedPoint(pub U128);
```

Comment:
Is the use of uint type required if we are using a type that has a primitive equivalent? I mean `U128` -> `u128`